app/cookies/RecurringContributionCookie.scala

```diff
@@ -0,0 +1,18 @@
+package cookies
+
+import java.time.Instant
+
+import com.gu.support.workers.model.BillingPeriod
+import play.api.mvc.Cookie
+
+object RecurringContributionCookie {
+
+  val currentTime = Instant.ofEpochSecond(System.currentTimeMillis / 1000).toString
+
+  def create(domain: String, billingPeriod: BillingPeriod): Cookie = Cookie(
+    name = "gu.contributions.recurring.contrib-timestamp." + billingPeriod,
```

Comment:
not much in it, but you could maybe use string interpolation here if you think it's any nicer `s"gu.contributions.recurring.contrib-timestamp.$billingPeriod"`